### PR TITLE
cut v0.4.0 release

### PR DESCRIPTION
**Pull request by M00nF1sh**
cut v0.4.0 release

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,22 @@
 # CHANGELOG
 
+## v0.4.0
+
+### Summary
+  This version supports EKS-Fargate pods and bumps the default envoy version to v1.12.2
+
+### Changes:
+* Copy attribution doc to image (#118, @nckturner)
```
Comment (nckturner) [inline CHANGELOG.md]: Can you just make these links to be consistent with the rest of the changelog?  If its annoying just write a script to generate it :D

Comment (M00nF1sh) [reply CHANGELOG.md]: :D

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,22 @@
...
+* Update envoy image version to v1.12.2.1-prod ([#115](https://github.com/aws/aws-app-mesh-inject/pull/115), @abaptiste)
+* Revert "Update Envoy image to v1.12.2.0-prod" ([#113](https://github.com/aws/aws-app-mesh-inject/pull/113), @nckturner)
+* Update Envoy image to v1.12.2.0-prod ([#112](https://github.com/aws/aws-app-mesh-inject/pull/112), abaptiste)
+* Support for Amazon EKS on AWS Fargate ([#111](https://github.com/aws/aws-app-mesh-inject/pull/111), @kiranmeduri)
+* Rename config dir to envoy-tracing-config ([#108](https://github.com/aws/aws-app-mesh-inject/pull/108), @nckturner)
+* Update default app mesh envoy sidecar to v1.12.1.1 ([#107](https://github.com/aws/aws-app-mesh-inject/pull/107), @lavignes)
+* Add AWS_REGION env-var to xray-daemon ([#106](https://github.com/aws/aws-app-mesh-inject/pull/106), @kiranmeduri)
+* Propogate additional settings via Pod to CNI in container runtime ([#105](https://github.com/aws/aws-app-mesh-inject/pull/105), @kiranmeduri)
+* Enable CNI to take over netfilter rules setup from init-container ([#104](https://github.com/aws/aws-app-mesh-inject/pull/104), @kiranmeduri)CHANGELOG.md
```
Comment (nckturner) [inline CHANGELOG.md]: nit extra `CHANGELOG.md` at the end of this line

Comment (M00nF1sh) [reply CHANGELOG.md]: nice catch :D

**Pull request merged**